src/data.rs

```diff
@@ -28,6 +28,12 @@ pub struct DataTree<'a> {
     raw: *mut ffi::lyd_node,
 }
 
+#[derive(Debug)]
+pub struct DataTreeOwningRef<'a> {
```

Comment:
nit: please add a doc comment for this struct

Reply:
done